views/home.hbs

```diff
@@ -1 +1,5 @@
-<h1>hey easyNaz</h1>
+<header class="tc">
+  <a href='/business'>
+    <img src="./assets/tempLogo.png" alt="">
```

Comment:
I know it will change in the future, but I think we should get into the habit of putting an "alt" attribute into all the images, for screenreaders.

```"alt"="almanarah logo"```